README.md

```diff
@@ -252,6 +252,17 @@ fileignoreconfig:
 ```
 Entering this in the `.talismanrc` file will ensure that Talisman will ignore the `danger.pem` file as long as the checksum matches the value mentioned in the `checksum` field.
 
+### Interactive mode
+If it is too much of a hassle to keep copying content to .talismanrc everytime you encounter an error from Talisman, you could enable the interactive mode and let Talisman assist you in prompting the additions of the files to ignore. 
+Just follow the simple steps:
+1. Open your bash profile where your environment variables are set (.bashrc, .bash_profile, .profile or any other location)
+2. You will see `TALISMAN_INTERACTIVE` variable under `# >>> talisman >>>`
+3. If not already set to true, add `export TALISMAN_INTERACTIVE=true`
+4. Don't forget to save and source the file
+
+That's it! Every time Talisman hook finds an error during pre-push/pre-commit, just follow the instructions as Talisman suggests. 
+Be careful to not ignore a file without verifying the content. You must be confident that no secret is getting leaked out.
+
```

Comment:
Best to add a note stating that IDE's that have git integration will fail in making commits if talisman is setup to run in interactive mode for the repository in question.

Reply:
@vhasus I have added details about Interactive mode to the readme file. Please check again now.